cmd/client.go

```diff
@@ -72,6 +74,12 @@ func logNodeBalance(store *strpkg.Store) {
 	logger.Infow(balance)
 }
 
+func logConfigVariables(config strpkg.Config, debug bool) {
+	if debug {
+		logger.Debug("Environment variables\n", config)
```

Comment:
`logger.Debug` will already not print if `debug` is false, so you can remove this `if` and the passed `debug bool`.